src/components/TransactionList/TransactionRow/TransactionRowTs.ts

```diff
@@ -220,9 +229,21 @@ export class TransactionRowTs extends Vue {
             .format(DateTimeFormatter.ofPattern('yyyy-MM-dd HH:mm:ss'));
     }
     public get date() {
-        return this.transaction.deadline
-            .toLocalDateTime(this.networkConfiguration.epochAdjustment)
-            .minusHours(2)
-            .format(DateTimeFormatter.ofPattern('yyyy-MM-dd HH:mm:ss'));
+        if (this.transaction instanceof AggregateTransaction) {
+            return this.transaction.deadline
+                .toLocalDateTime(this.networkConfiguration.epochAdjustment)
+                .minusHours(48)
```

Comment:
Maybe it's better to keep these values in a config file?

Reply:
@OlegMakarenko we're discussing changing the deadline values for aggregates and hashlocks in a different pr, once we come up with a decision will move them to configs